tox.ini

```diff
@@ -11,6 +11,7 @@ envlist =
     py311
     pypy3
     py37-{pexpect,xdist,unittestextras,numpy,pluggymain}
+    py311-exceptiongroup
```

Comment:
This new tox environment needs to be added to `.github/workflows/test.yml`, otherwise it won't execute in CI.

Reply:
added on ubuntu, do you think it's worth testing on windows too?
I'm also curious why py311 is not run with `xdist`, I defaulted to follow suit and not do as well.

Reply:
I also took the liberty of adding `use_coverage` to ubuntu-py311 to get 100% code coverage